components/infobox/wikis/stormgate/infobox_building_custom.lua

```diff
@@ -227,11 +251,11 @@ function CustomBuilding:_getDefenseDisplay()
 	local armor = tonumber(args.armor)
 
 	return table.concat(Array.append({},
-		health and ICON_HP or nil,
-		health,
+		ICON_HP,
+		health or 0,
 		extraHealth and ('(+' .. extraHealth .. ')') or nil,
-		armor and ICON_ARMOR or nil,
-		armor
+		ICON_ARMOR,
+		armor or 0
```

Comment:
these changes were already pushed with #4528

Reply:
#4528 was for unit navbox (the title was somehow changed to "building"), this pr is follow-up for building navbox I didn't want to include in #4258 since this was a quick bu fix pr.